src/services/functionAppService.ts

```diff
@@ -40,17 +41,10 @@ export class FunctionAppService extends BaseService {
 
   public async getMasterKey(functionApp?: Site) {
     functionApp = functionApp || await this.get();
-    const adminToken = await this.getAuthKey(functionApp);
```

Comment:
why is adminToken not require anymore?

Reply:
admin token is overkill. you only need a key for that function app